packages/wp-dashboard/src/components/editorSettings/telemetry/index.js

```diff
@@ -64,7 +64,7 @@ export default function TelemetrySettings({
             <TranslateWithMarkup
               mapping={{
                 a: (
-                  //eslint-disable-next-line jsx-a11y/anchor-has-content
+                  //eslint-disable-next-line jsx-a11y/anchor-has-content -- @todo
```

Comment:
This is simply a false positive.

I think `jsx-a11y/anchor-has-content` doesn't add much value in our code base.

Let's disable that rule in our ESLint config and remove this comment.